Fix failure on first paste from CLIPBOARD

uli42 opened the pull request:
Pasting the clipboard did not work directly after session start.

I am a bit what other effects might be introduced by leaving out this code. Maybe we should find out why it was introduced in the first place...

sunweaver commented on nx-X11/programs/Xserver/hw/nxagent/Clipboard.c:
```diff
@@ -1583,6 +1583,9 @@ int nxagentInitClipboard(WindowPtr pWin)
 
+  /*
+     Removing this seems to solve xg2o issue #952 (no paste directly after session start)
+
```
Thanks for providing this patch. Please be less X2Go BTS centric in the complete PR. Describe what your patch fixes:
- in the PR name (please rename)
- in the comment above
- in the commit msg

sunweaver commented:
Please mention your concerns as given in the PR text in the comment above the commented out code passage. So we know, that we might expect other weird behaviour from commenting out the discovered code passage. Please fix typos/grammar in that text, too.

In the merge, I will reference this PR number and the X2Go BTS number (in a machine readable way). No need to mention the X2Go BTS all over the place. Please mention the X2Go BTS number in the comment-in-the-code, but only at the end of your textual explanation.

E.g.: This passage breaks first paste on Windows, whereas being unsure what other effects might be introduced by leaving out this code. (See https://bugs.x2go.org/952 for details).

Thanks!
Mike

uli42 commented:
OK, I have updated the commit as requested.

I also did some research and found that this code had emerged between nxagent 1.4.0-33 and 1.4.0-47 (I do not have any intermediate releases). However, the changelog only contains two commits regarding clipboard:

```
nxagent-1.4.0-35

- Implemented a new method to comunicate to nxclient,
  the raise of the agent root window, taking the ownership
  of the selection "A-{MD5 of session}".
  Used the same selection to let nxclient comunicate to agent
  by changing the property on the same string, when the user
  choose by the administrator to terminate or suspend the
  session.
```

and

```
nxagent-1.4.0-44

- Fixed a problem in the clipboard procedure.
  Now when we get a request of the selection
  from an internal client we have to, if the 
  owner is on the X server, forward the request 
  to X, otherwise we have to pass the request 
  to our internal client.
  But for a problem in this procedure we passed, 
  in some situation, the request to the internal 
  client even if the owner was on the other side.
```

None of them seems to address the lines I have commented in this commit...
